air-backend/installers/community/amd64/air-backend/export.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+mkdir -p ./archives
+
+source .env
+
+docker-compose pull || exit 1
+
+docker save --output ./archives/dgraph-${DGRAPH_VERSION}.tar dgraph/dgraph:${DGRAPH_VERSION} || exit 1
+
+docker save --output ./archives/eclipse-mosquitto-${ECLIPSE_MOSQUITTO_VERSION}.tar eclipse-mosquitto:${ECLIPSE_MOSQUITTO_VERSION} || exit 1
+
+docker save --output ./archives/labs-air-ui-oss-${LABS_AIR_VERSION}.tar public.ecr.aws/tibcolabs/labs-air-ui-oss:${LABS_AIR_VERSION} || exit 1
+
+docker save --output ./archives/labs-air-data-mqtt-dgraph-${LABS_AIR_VERSION}.tar public.ecr.aws/tibcolabs/labs-air-data-mqtt-dgraph:${LABS_AIR_VERSION} || exit 1
+
+docker save --output ./archives/labs-air-metadata-mqtt-dgraph-${LABS_AIR_VERSION}.tar public.ecr.aws/tibcolabs/labs-air-metadata-mqtt-dgraph:${LABS_AIR_VERSION} || exit 1
+
+docker save --output ./archives/labs-air-notification-mqtt-dgraph-${LABS_AIR_VERSION}.tar public.ecr.aws/tibcolabs/labs-air-notification-mqtt-dgraph:${LABS_AIR_VERSION} || exit 1
+
+docker save --output ./archives/labs-air-pipeline-service-${LABS_AIR_VERSION}.tar public.ecr.aws/tibcolabs/labs-air-pipeline-service:${LABS_AIR_VERSION} || exit 1
+
```

Comment:
If this file can utilize the `scripts/functions.sh` file, it could replace the `|| exit 1` statements with a `checkError ${?} "My informational exit statement"` This will print a stack trace that could help the user find and resolve any errors.

Reply:
I have created a task to implement this in a different step: https://app.clickup.com/t/2w31056
We need to have this installer ASAP.